src/tsconfig-base.json

```diff
@@ -12,14 +12,16 @@
         "composite": true,
         "noEmitOnError": true,
 
-        "strictNullChecks": true,
-        "noImplicitAny": true,
-        "noImplicitThis": true,
-        "strictPropertyInitialization": true,
+        "strict": true,
         "noUnusedLocals": true,
         "noUnusedParameters": true,
+        "allowUnusedLabels": false,

```

Comment:
Isn't this false by default?

Reply:
The default is `undefined` which just shows warnings in editors. https://www.typescriptlang.org/tsconfig#allowUnusedLabels

I was pulling rules from https://www.npmjs.com/package/@tsconfig/node16-strictest, for reference.